src/__tests__/app.integration.test.js

```diff
@@ -92,10 +87,10 @@ describe('Swagger Documentation', () => {
 });
 
 describe('GET /registration-requests/:conversationId', () => {
-  const conversationId = v4();
+  const conversationId = v4(); // OutboundConversationId
```

Comment:
Could we rename this variable please

```suggestion
  const outboundConversationId = v4();
```